subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2512,7 +2509,9 @@ static int handle_request(struct coap_packet *request,
 		break;
 
 	case COAP_METHOD_PUT:
-		context.operation = LWM2M_OP_WRITE;
+		/* write attributes if content-format is absent */
+		context.operation = format == LWM2M_FORMAT_NONE ?
```

Comment:
I know it's not necessary but can we add parentheses to clarify the operations here.